src/_includes/layouts/events.njk

```diff
@@ -0,0 +1,88 @@
+---
+layout: layouts/fullwidth.njk
+---
+
+{% from "../../../node_modules/govuk-frontend//dist/govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
+{{ govukBreadcrumbs({
+  items: [
+    {
+      text: site.name,
+      href: "/"
+    },
+    {
+      text: domain,
+      href: "/domains/"+domain
+    },
+        {
+      text: "Events",
+      href: "/"+domain+'/events'
+    },
```

Comment:
This page doesn't exist, but you could remove this breadcrumb if you add the events to the domain page (see other comment).